manifests/P/PlasmoApp/plasmo_voice/main.yaml

```diff
@@ -0,0 +1,23 @@
+manifestSpecVersion: "4.0"
+publisher: PlasmoApp
+iconUrls:
+  - https://cdn.modrinth.com/data/1bZhdhsH/icon.png
+  - https://raw.githubusercontent.com/plasmoapp/plasmo-voice/client-1.18-1.2.x/common/src/main/resources/assets/plasmo_voice/icon.png
+status: active
+updatedAlternatives: ~
+name: "Plasmo Voice"
+description: A proximity voice chat mod with audio positioning and lots of features.
+authors:
+  - name: kpids
+  - name: Apehum
```

Comment:
Yes, while they did make the mod, the `fabric.mod.json` only says `Plasmo`, so you should only put `Plasmo` under the authors.